src/smclarify/bias/metrics/posttraining.py

```diff
@@ -13,6 +13,7 @@
 import numpy as np
 from sklearn.neighbors import KNeighborsClassifier
 from smclarify.bias.metrics.constants import INFINITY, FT_DEFAULT_NEIGHBOR, FT_MIN_NEIGHBOR, FT_SAMPLES_COUNT_THRESHOLD
+from typing import List
```

Comment:
this change is not needed, you can remove it.

Reply:
good catch!